### PR TITLE
ODFE 1.13.0: Schedule check plugin workflow and change manifest versions

### PR DESCRIPTION
**Pull request by sreekarjami**
*Issue #, if available:*

*Description of changes:*
schedule check plugin workflow and change manifest as per the new release

*Test Results:*
No test needed. 

**Note: If this PR is related to Helm, please also update the README for related documentation changes. Thanks.**
**https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/master/helm/README.md**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/staging-copy-artifacts.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Staging Copy Artifacts
 
 on:
 #  schedule:
-#    - cron: '30 10 * * *' 
+#    - cron: '0 9 * * *'
```
Comment (peterzhuamazon) [inline .github/workflows/staging-copy-artifacts.yml:5]: Why change the time from 10:30 UTC to 9:00 UTC?

**Review by gaiksaya — APPROVED** [review]

File: .github/workflows/staging-copy-artifacts.yml (hunk 1 above)
Comment (sreekarjami) [reply .github/workflows/staging-copy-artifacts.yml:5]: As the building workflows are set to start at 10. So I moved RC to 9 UTC.

**Comment by peterzhuamazon** [comment]
You can assume all plugins are having 1.13.0.0 from the start (edited) 
Since all of them will for sure have its initial version as 1.13.0.0

**Comment by sreekarjami** [comment]
Made changes to manifest to set all plugins version to 1.13.0.0

**Review by peterzhuamazon — APPROVED** [review]
Thanks

**Pull request merged**